.eslintrc

```diff
@@ -5,7 +5,8 @@
   },
   "rules": {
     "brace-style": off,
-    "padded-blocks": off
+    "padded-blocks": off,
+    "lines-around-comment": ["error", { "beforeLineComment": true, "beforeBlockComment": true, "allowBlockStart": false }]
```

Comment:
Enforces stylistic feedback received earlier by failing build on missing newlines before comments at the start of blocks.